### PR TITLE
Changed Readme and Added Euler circuit and Path

### PR DESCRIPTION
**Pull request by atriadhiakri2000**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ Y ] I've read the contribution guidelines.
- [ Y ] I've checked the issue list before deciding what to submit.
- [ Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by atriadhiakri2000** [comment]
@iamrajiv @shraddhavp @HarshCasper  Please check my PR

File: Python/graphs/Euler_circuit_and_Path.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#MODULES
+
+# Importing sys module to initalize the maxsize
+import sys
+# Importing collections model to access deque
+from collections import deque
```
Comment (atarax665) [inline Python/graphs/Euler_circuit_and_Path.py]: You can import both ```deque and defaultdict``` at once using ```from collections import deque, defaultdict```

**Review by atarax665 — CHANGES_REQUESTED** [review]
The code looks good to me, I would like you to just format the code according to our [python code template](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/Code_Template.md) to maintain the code consistency in this repository.

**Comment by atriadhiakri2000** [comment]
@iamrajiv @shraddhavp @HarshCasper Please check my PR

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Comment by shraddhavp** [comment]
conflicts to be resolved @atriadhiakri2000

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**